sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Constants.cs

```diff
@@ -9,6 +11,10 @@ internal static class Constants
         public const string DefaultConnectionSettingStringName = "AzureWebJobsServiceBus";
         public const string DynamicSku = "Dynamic";
         public const string AzureWebsiteSku = "WEBSITE_SKU";
+        public const string TargetBasedScalingEnabled = "TARGET_BASED_SCALING_ENABLED";
+        public const string DynamicConcurrencyEnabled = "DYNAMIC_CONCURRENCY_ENABLED";
```

Comment:
We do not need to depend on these app settings anymore, aside from `TARGET_BASED_SCALING_ENABLED`. Dynamic concurrency integration is used only when the following settings are present in a function app's host.json:

```{ 
        "version": "2.0", 
        "concurrency": { 
            "dynamicConcurrencyEnabled": true, 
            "snapshotPersistenceEnabled": true 
        } 
    }
```
Additionally, the targetMetric value is determined by the `maxConcurrentCalls` settings (if present) in a user's host.json

See the variables used in target based scaling [here](https://msazure.visualstudio.com/One/_git/AAPT-Antares-ScaleController?path=%2F&version=GBv2.x&_a=contents#:~:text=summary%20for%20details.-,Target%20Based%20Scaling,-Algorithm%20Overview).